src/syft/core/node/common/node.py

```diff
@@ -262,8 +262,10 @@ def __init__(
     def icon(self) -> str:
         return "📍"
 
-    def get_client(self, routes: List[Route] = []) -> ClientT:
-        if not len(routes):
+    def get_client(self, routes: List[Route] = None) -> ClientT:
```

Comment:
We run mypy with `--no-implicit-optional` so this will probably fail in our linter step.
I think the best way to refactor this would be to introduce an Optional or just remove the two if statements since they are not needed.